aten/src/ATen/Declarations.cwrap

```diff
@@ -316,6 +316,8 @@
 ]]
 [[
   name: _th_equal
+  cpu_bool: True
+  cuda_bool: True
```

Comment:
You will have to either remove cuda_bool or add cuda related changes, which is more preferable.

Reply:
Done, CUDA is a bit harder to test for me.